contracts/modules/governance/MemberRoles.sol

```diff
@@ -223,14 +233,16 @@ contract MemberRoles is IMemberRoles, Governed, LegacyMasterAware {
     uint[][] calldata stakingPoolTokenIds
   ) external override {
     _switchMembership(msg.sender, newAddress);
-    nxm.transferFrom(msg.sender, newAddress, nxm.balanceOf(msg.sender));
+    INXMToken _token = token();
+    _token.transferFrom(msg.sender, newAddress, _token.balanceOf(msg.sender));
 
+    ICover _cover = cover();
     // Transfer the cover NFTs to the new address, if any were given
-    cover.transferCovers(msg.sender, newAddress, coverIds);
+    cover().transferCovers(msg.sender, newAddress, coverIds);
```

Comment:
`cover()` is called twice here. i think you meant to use the `_cover` variable instead?